packages/snage/src/command/create.ts

```diff
@@ -1,11 +1,10 @@
 import yargs from 'yargs';
```

Comment:
Erlaubt noch invalide enum typen, wird aber schatze ich mal gefixt, wenn die validatoren entfernt werden und durch die fürs note parsing ersetzt werden:
```
jm@jannix changelog/packages/snage cli-tool-improvements✗ λ yarn dev --config ../../.snage.yaml create --components jesus --components test
yarn run v1.22.4
$ ts-node src/index.ts --config ../../.snage.yaml create --components jesus --components test
? Please enter value for issue #12
? Select type change
? Select audience user
? [OPTIONAL] Please enter value for version
? Do you want to set the current date for date? Yes
Successfully created file: changelog/#12-change.md
true
Done in 10.17s.
jm@jannix changelog/packages/snage cli-tool-improvements✗ λ cat changelog/\#12-change.md
---
issue: "#12"
type: change
audience: user
date: 2020-05-05
components:
  - jesus
  - test
---
```

Reply:
Jau, das kommt mit https://github.com/FACT-Finder/snage/issues/50
Aktuell hat man ja auch in zwei Dateien ähnliche Validierungscalls + Parsing etc, das lässt sich sicher geschickter zusammen fassen, würde aber imo langsam den Rahmen vom PR sprengen, das würde sonst wieder so groß werden